earth_enterprise/src/common/tests/SConscript

```diff
@@ -50,8 +51,12 @@ map(lambda test: env.test(test, [test + '.cc'],
     ['etencoder_tests',
      ])
 
+testdir = os.path.join(env.exportdirs['bin'],'tests')
+env.copyfile(testdir, 'RunAllTests.pl')
 
-env.copyfile(os.path.join(env.exportdirs['bin'],'tests'),
-             'RunAllTests.pl')
+# Drop in the test data
+fusiontestdir = os.path.join(testdir, 'fusion')
+shutil.rmtree(fusiontestdir, ignore_errors=True)
+shutil.copytree('../../fusion', fusiontestdir)
```

Comment:
If you run from the source root, you'll end up with some test products in the source dir (rather than the build dir). That is messy (pollutes git, which could be overcome with .gitignore) and might be an issue for building on multiple architectures when we get to that stage.